.github/actions/upload-log/action.yml

```diff
@@ -0,0 +1,25 @@
+name: Upload the log file while running
+
+description: Upload the log file while the job runs
+
+inputs:
+  job-id:
+    description: Job id
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Upload log file in background
+      shell: bash
+      continue-on-error: true
+      env:
+        JOB_ID: ${{ inputs.job-id }}
+      run: |
+        set -x
+        # Complete job name: shows up in the Set up job step
+        file_name=$(grep -R "Complete job name:" "${{ runner.workspace }}/../../_diag/pages" -l)
+        while true; do
+          aws s3 cp "${file_name}" "s3://ossci-raw-job-status/log/${JOB_ID}" --content-type "text/plain"
```

Comment:
Using `aws s3 sync` would be a better choice here, to avoid re-uploading the file if the content has been changed

Reply:
apparently sync doesnt work in this case because it expects 2 folders

Reply:
I see, so the command line doesn't support that.  Using boto with a bit of coding, you you actually sync a file instead of the whole directory.  But that's overkill here, so let's stick with cp